apis/v1alpha1/gatewayclass_types.go

```diff
@@ -118,7 +118,8 @@ const (
 type GatewayClassStatus struct {
 	// Conditions is the current status from the controller for
 	// this GatewayClass.
-	// +optional
+	//
+	// +kubebuilder:validation:MaxItems=8
```

Comment:
Same as https://github.com/kubernetes-sigs/service-apis/pull/336/files#r497816693

Reply:
I'm not sure what to change here. I don't think this should be optional here given the default value. Are there other differences I'm missing?